oger/doc/txt.py

```diff
@@ -141,3 +142,17 @@ def iter_documents(self, source):
                     continue  # member is not a regular file
                 id_ = basename(member.name)
                 yield self._document(stream, id_)
+
+
+class TXTTSVLoader(DocIterator, _TXTLoaderMixin):
+    '''
+    Loader for single TSV document consisting of 
+    ColumnA -> [id]
+    ColumnB -> [text] format
+    '''
+    def iter_documents(self, source):
+        with text_stream(source) as f:
+            data = csv.reader(f, dialect='excel', delimiter='\t')
+            next(data) # skips header row
+            for id_, text in data:
+                yield self._document([text], id_)
```

Comment:
```suggestion
                yield self._document(iter([text]), id_)
```
`self._document` expects an iterator and results in text duplication if provided a collection.